tests/test_client.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import requests
+import aiohttp
+
+from unittest.mock import MagicMock, AsyncMock
+from sdc_client import client
+from sdc_client.base_api_client import UnauthorizedException
+from conftest import MockResponse, MockAsyncResponse, StreamSetsMock
+
+
+class TestGetJMX(unittest.TestCase):
+    def test_get_jmx_success(self):
+        requests.Session.get = MagicMock(return_value=MockResponse(
+            _text='data',
+            status_code=200
+        ))
+        result = client.get_jmx(StreamSetsMock(), 'query_params_1')
+        self.assertEqual(result, {'json': 'data'})
+
+    def test_get_jmx_unauthorized(self):
+        requests.Session.get = MagicMock(return_value=MockResponse(
+            _text='data',
+            status_code=401,
+        ))
+        with self.assertRaises(UnauthorizedException):
+            client.get_jmx(StreamSetsMock(), 'query_params_1')
+
+    def test_get_jmx_async_success(self):
+        aiohttp.ClientSession.get = AsyncMock(return_value=MockAsyncResponse(
+            _text='data',
+            status_code=200
+        ))
+        queries = [
+            (StreamSetsMock(), 'query_params_1',),
+            (StreamSetsMock(), 'query_params_2',),
+        ]
+        result = client.get_jmxes_async(queries=queries)
+        self.assertEqual(result, [{'json': 'data'}, {'json': 'data'}])
+
+    def test_get_jmx_async_unauthorized(self):
+        aiohttp.ClientSession.get = AsyncMock(return_value=MockAsyncResponse(
+            _text='data',
+            status_code=401
+        ))
+        queries = [
+            (StreamSetsMock(), 'query_params_1',),
+            (StreamSetsMock(), 'query_params_2',),
+        ]
+        with self.assertRaises(UnauthorizedException):
+            client.get_jmxes_async(queries=queries)
```

Comment:
to run the test please add 
```
if __name__ == '__main__':
    unittest.main()
```
also please add the same to test_balancer.py, for some reason it's missing there
or run tests using pytest lib